IoTConnect_2.0/IoTConnect_2.0/IoTConnectManager/IoTConnectManager.swift

```diff
@@ -726,24 +771,61 @@ class IoTConnectManager {
         }
     }
     
+    //get twins
     func getTwins(callBack: @escaping GetTwinCallBackBlock) -> () {
         if dictSyncResponse.count > 0 {
-            objMQTTClient.publishTopicOnMQTT(withData:["mt":CommandType.GET_DEVICE_TEMPLATE_TWIN], topic: "")
+            objMQTTClient.publishTopicOnMQTT(withData:["mt":CommandType.GET_DEVICE_TEMPLATE_TWIN.rawValue], topic: "")
         }else{
             objCommon.manageDebugLog(code: Log.Errors.ERR_GA03, uniqueId: strUniqueId, cpId: strCPId, message: "", logFlag: false, isDebugEnabled: boolDebugYN)
             callBack("Twins data not found")
         }
     }
     
+    //get child devices
     func getChildDevices(callBack: GetChildDevicesCallBackBlock) -> () {
         if dictSyncResponse.count > 0 {
-            objMQTTClient.publishTopicOnMQTT(withData:["mt":CommandType.GET_CHILD_DEVICE], topic: "")
+            objMQTTClient.publishTopicOnMQTT(withData:["mt":CommandType.GET_CHILD_DEVICE.rawValue], topic: "")
         }else{
             objCommon.manageDebugLog(code: Log.Errors.ERR_GA04, uniqueId: strUniqueId, cpId: strCPId, message: "", logFlag: false, isDebugEnabled: boolDebugYN)
             callBack("Child Devices data not found")
         }
     }
     
+    //called on data frequncy change
+    func onFrequencyChangeCommand(dfValue:Int){
+        var metaInfo = self.dictSyncResponse["meta"] as? [String:Any]
+        metaInfo?["df"] = dfValue
+//        print("metaInfo Before \(self.dictSyncResponse)")
+        self.dictSyncResponse["meta"] = metaInfo
+//        print("metaInfo after \(self.dictSyncResponse)")
+        df = metaInfo?["df"] as? Int ?? 0
+        print("df changed val \(df)")
+    }
+    
+    //Publish topic on MQTT to create child device
+    public func createChildDevice(deviceId:String, deviceTag:String, displayName:String){
+        let metaInfo = self.dictSyncResponse["meta"] as? [String:Any]
+        let gtw = metaInfo?["gtw"] as? [String:Any]
+        let g = gtw?["g"] as? String
+        
+        objMQTTClient.publishTopicOnMQTT(withData: ["mt":CommandType.CREATE_DEVICE.rawValue,
```

Comment:
Same comment here

Reply:
@sw-ot-deveshmevada Changes are done in new push